api/post.go

```diff
@@ -486,53 +486,27 @@ func getExplicitMentions(message string, keywords map[string][]string) (map[stri
 		}
 	}
 
-	for _, word := range strings.Fields(message) {
-		isMention := false
+	splitWords := strings.FieldsFunc(message, func(c rune) bool {
+		return model.SplitRunes[c]
+	})
 
-		if word == "@here" {
+	for _, splitWord := range splitWords {
```

Comment:
Does this still let you mention users who's names contain a period, underscore, or hyphen? That was the reason we had that weird, two-step splitting process